packages/components/src/mobile/audio-player/index.native.js

```diff
@@ -27,22 +32,23 @@ const isIOS = Platform.OS === 'ios';
 
 function Player( {
 	getStylesFromColorScheme,
-	source,
 	isUploadInProgress,
 	fileName,
 	isUploadFailed,
+	attributes,
+	isSelected,
 } ) {
 	const onPressListen = () => {
-		if ( source ) {
-			Linking.canOpenURL( source )
+		if ( attributes.src ) {
+			Linking.canOpenURL( attributes.src )
```

Comment:
Hi @ceyhun can I do inline object restructuring here instead of using `attributes.` I am thinking to try that.

Reply:
```js
const { src } = attributes;
if ( src ) {
	Linking.canOpenURL( src )
```
Something like this should work 👍

Reply:
Went ahead and did this here: [`a97bb19` (#28540)](https://github.com/WordPress/gutenberg/pull/28540/commits/a97bb19bdc17cb333ed24a18869516af63669b92)

Reply:
Beautifully done. Thanks 🙇